src/dialogs/more-info/controls/more-info-vacuum.ts

```diff
@@ -0,0 +1,286 @@
+import "@polymer/iron-icon/iron-icon";
+import "@polymer/paper-icon-button/paper-icon-button";
+import "@polymer/paper-item/paper-item";
+import "@polymer/paper-listbox/paper-listbox";
+
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+
+import { supportsFeature } from "../../../common/entity/supports-feature";
+import { HomeAssistant } from "../../../types";
+
+import "../../../components/ha-paper-dropdown-menu";
+import "../../../components/ha-attributes";
+import {
+  VACUUM_SUPPORT_BATTERY,
+  VACUUM_SUPPORT_CLEAN_SPOT,
+  VACUUM_SUPPORT_FAN_SPEED,
+  VACUUM_SUPPORT_LOCATE,
+  VACUUM_SUPPORT_PAUSE,
+  VACUUM_SUPPORT_RETURN_HOME,
+  VACUUM_SUPPORT_START,
+  VACUUM_SUPPORT_STATUS,
+  VACUUM_SUPPORT_STOP,
+  VacuumEntity,
+} from "../../../data/vacuum";
+
+class VacuumCommand {
```

Comment:
Seems unnecessary to create a class for this, can we just make it an typed object?

Reply:
I changed the class to an interface. I was not sure if this is your intention, but I'm new in typescript and this I found on google for typed object. If not, please give me an example